.gitignore

```diff
@@ -3,6 +3,8 @@
 /coverage/
 /demos/*/package-lock.json
 /demos/*/node_modules/
+/demos/*/tmp/
+/demos/bundlers/test-*.html
```

Comment:
Couldn't this lie in `tmp` as well so that the previous line is enough?

Reply:
Yep, I've done so already actually, but I forgot to remove the left-over line from gitignore. Fixed!